extra/2-piping.js

```diff
@@ -1,41 +1,49 @@
-/*
-  PIPING FUNCTIONS
-  ================
-  1. Write 3 functions:
-  - one that adds 2 numbers together
-  - one that multiplies 2 numbers together
-  - one that formats a number so it's returned as a string with a £ sign before it (e.g. 20 -> £20)
-
-  2. Using the variable startingValue as input, perform the following operations using your functions all
-  on one line (assign the result to the variable badCode):
-  - add 10 to startingValue
-  - multiply the result by 2
-  - format it
-  
-  3. Write a more readable version of what you wrote in step 2 under the BETTER PRACTICE comment. Assign
-  the final result to the variable goodCode
-*/
-
-function add() {
-
+/**
+ * /*
+ *   PIPING FUNCTIONS
+ *   ================
+ *   1. Write 3 functions:
+ *   - one that adds 2 numbers together
+ *   - one that multiplies 2 numbers together
+ *   - one that formats a number so it's returned as a string with a £ sign before it (e.g. 20 -> £20)
+ *
+ *   2. Using the variable startingValue as input, perform the following operations using your functions all
+ *   on one line (assign the result to the variable badCode):
+ *   - add 10 to startingValue
+ *   - multiply the result by 2
+ *   - format it
+ *
+ *   3. Write a more readable version of what you wrote in step 2 under the BETTER PRACTICE comment. Assign
+ *   the final result to the variable goodCode
+ *
+ * @format
+ */
+
+function add(num1, num2) {
+	var result = num1 + num2;
+	return result;
 }
 
-function multiply() {
-
+function multiply(num1, num2) {
+	var result = num1 * num2;
+	return result;
 }
 
-function format() {
-
+function format(num) {
+	var result = `£${num}`;
+	return result;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
-
+let badCode = add(startingValue, 10);badCode = multiply(badCode, startingValue);badCode = format(badCode);
+//Code is very difficult to read when it is written on single line, makes it challenging to rectify errors if encountered.
 /* BETTER PRACTICE */
 
-let goodCode =
+let goodCode = add(startingValue, 10);
+goodCode = multiply(goodCode, startingValue);
+goodCode = format(goodCode);
```

Comment:
This is much better. I think it could be even easier to understand if you declare a new variable for each line, describing what had been done, i.e. valueAfterAddition. But this is just a personal preference, your way is fine